public/index.html

```diff
@@ -258,6 +256,7 @@
     <section>
```

Comment:
you're definitely missing some of my changes here. Did you merge mainline into this branch? If there were conflicts you may have resolved them incorrectly.  this `<section></section>` should look like:
     <section class="mapArea">
     </section>